R/otel.R

```diff
@@ -0,0 +1,66 @@
+otel_tracer_name <- "org.r-dbi.DBI"
+
+# Generic otel helpers:
+
+otel_cache_tracer <- NULL
+otel_local_active_span <- NULL
+
+local({
+  otel_tracer <- NULL
+  otel_is_tracing <- FALSE
+
+  otel_cache_tracer <<- function() {
+    requireNamespace("otel", quietly = TRUE) || return()
```

Comment:
What happens if otel is installed during the session? Can we somehow support this use case?

Will otel print diagnostics on the console if it's active, by default?